RELEASE_NOTES.md

```diff
@@ -116,13 +116,11 @@ bin/rails decidim:upgrade:fix_orphan_categorizations
 
 You can read more about this change on PR [\#12143](https://github.com/decidim/decidim/pull/12143).
 
-### 3.5. [[TITLE OF THE ACTION]]
+### 3.5. Clean up orphaned attachment blobs
 
-You can read more about this change on PR [\#XXXX](https://github.com/decidim/decidim/pull/XXXX).
-
-### 3.13 Clean up orphaned attachment blobs
+We have added a new task that helps you clean the orphaned attachment blobs. This task will remove all the attachment blobs that have been created for more than 1 hour and are not yet referenced by any attachment record. This helps cleaning your filesystem of unused files.
 
-We have added a new task that helps you clean the orphaned attachment blobs. This task will remove all the attachment blobs that are not referenced by any attachment record. This helps cleaning your filesystem of unused files.
+For a consistent behavior we are recommending to run this task 2 times at least 1 hour apart.
 
```

Comment:
This is actually not needed, as I was being overly paranoid and it doesn't apply the attack that I had in mind anymore 

```suggestion
```